examples/ipython/WORKSPACE

```diff
@@ -0,0 +1,48 @@
+workspace(name = "rules_py_ipython_example")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+local_repository(
```

Comment:
This kinda bugs me - I'd like these examples to be standalone, allowing someone to take the directory as a starting point for a project, but I also don't want to have to maintain the `http_archive` alternative here.